packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,34 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+export const StyledBreadcrumbItem = styled.li.attrs(() => ({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
+  'data-garden-version': PACKAGE_VERSION
 }))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  ${props => `
+    line-height: ${(props.theme.space.base * 5) / stripUnit(props.theme.fontSizes.md)}}
+    color: ${props.isCurrent ? getColor(props.theme.colors.neutralHue, 600) : 'inherit'}
+    & a {
+      &:any-link {
+        color: ${props.isCurrent ? 'inherit' : ''};
+      }
+      &:any-link:hover {
+        color: ${props.isCurrent ? 'inherit' : ''};
+      }
+      &:any-link:focus {
+        color: ${props.isCurrent ? 'inherit' : ''};
+      }
+    }
+  `}
+  white-space: nowrap;
+  font-size: inherit;
```

Comment:
@hzhu once this is cleaned up based on existing comments, let's get together and review CSS property ordering for styled components. Thanks :pray:

Reply:
@jzempel that sounds great! I've re-ran the CSS lint and re-ordered CSS according to [stylelint-order](https://github.com/zendeskgarden/stylelint-config/blob/master/plugins/stylelint-order.js).

<img width="996" alt="Screen Shot 2019-11-30 at 5 45 15 PM" src="https://user-images.githubusercontent.com/1811365/69908197-7dd11580-1399-11ea-8248-e474b8fc8aec.png">